cmd/api.go

```diff
@@ -29,15 +29,13 @@ func init() {
 	rootCmd.AddCommand(balances)
 	rootCmd.AddCommand(issuance)
 	rootCmd.AddCommand(status)
-	rootCmd.AddCommand(burn)
 	rich.Flags().Int("count", 100, "The top X address")
 	rootCmd.AddCommand(rich)
 
 	get.AddCommand(getTX)
 	get.AddCommand(getRates)
 	getBank.Flags().Bool("raw", false, "Print the full json data")
 	get.AddCommand(getBank)
-	getTXs.Flags().Bool("burn", false, "Show burns")
```

Comment:
Burns are still part of the historical set of transactions and you should still be able to filter for them.